### PR TITLE
Update risingwave-docker-image.md

### PR DESCRIPTION
**Pull request by CharlieSYH**
## Info
Rn, we provide two same commands.
image

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

File: versioned_docs/version-0.1.17/deploy/risingwave-docker-image.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ Run the following command to pull and start the latest release of RisingWave in
 ```shell
 docker run -it --pull=always -p 4566:4566 -p 5691:5691 risingwavelabs/risingwave:latest playground
 ```
-> You can find the previous releases and nightly builds on [Docker Hub](https://hub.docker.com/r/risingwavelabs/risingwave/tags).<br/>For example, if you would like to use RisingWave `latest`, run:
+> You can find the previous releases and nightly builds on [Docker Hub](https://hub.docker.com/r/risingwavelabs/risingwave/tags).<br/>For example, if you would like to use RisingWave v0.1.16, run:
```
Comment (CAJan93) [inline versioned_docs/version-0.1.17/deploy/risingwave-docker-image.md]:
```suggestion
> You can find the previous releases and nightly builds on [Docker Hub](https://hub.docker.com/r/risingwavelabs/risingwave/tags).<br/>For example, if you would like to use RisingWave v0.1.17, run:
```

File: versioned_docs/version-0.1.17/deploy/risingwave-docker-image.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ Run the following command to pull and start the latest release of RisingWave in
...
 ```shell
-docker run -it --pull=always -p 4566:4566 -p 5691:5691 risingwavelabs/risingwave:latest playground
+docker run -it --pull=always -p 4566:4566 -p 5691:5691 risingwavelabs/risingwave:v0.1.16 playground
```
Comment (CAJan93) [inline versioned_docs/version-0.1.17/deploy/risingwave-docker-image.md]:
```suggestion
docker run -it --pull=always -p 4566:4566 -p 5691:5691 risingwavelabs/risingwave:v0.1.17 playground
```

Let's use the latest image that we have at the moment 😁

**Review by CAJan93 — APPROVED** [review]
Nit change. Otherwise LGTM. Thanks for mentioning the nightly images.

File: versioned_docs/version-0.1.17/deploy/risingwave-docker-image.md (hunk 2 above)
Comment (CharlieSYH) [reply versioned_docs/version-0.1.17/deploy/risingwave-docker-image.md]: Oh right! This is actually the versioned doc for v0.1.17, so your edit is on point.

**Comment by CharlieSYH** [comment]
> Nit change. Otherwise LGTM. Thanks for mentioning the nightly images.

Thanks, I made edits to other versions as well.

**Comment by hengm3467** [comment]
@CharlieSYH I suggest we use a variable to replace the actual versions. Otherwise, we need to make change to this one for each release, which sounds unnecessary.

**Comment by hengm3467** [comment]
> @CharlieSYH I suggest we use a variable to replace the actual versions. Otherwise, we need to make change to this one for each release, which sounds unnecessary.

Let's make the change when we do the versioning for the upcoming version 0.1.18.

**Pull request merged**